recipe/build.sh

```diff
@@ -1,32 +1,47 @@
 #!/usr/bin/env bash
-# Get an updated config.sub and config.guess
-cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils
+
+# Get an updated config.sub and config.guess 
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./CoinUtils 
 cp $BUILD_PREFIX/share/gnuconfig/config.* .
+
 set -e
 
-UNAME="$(uname)"
-export CFLAGS="${CFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS} -O3"
-export CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++11}"
+if [[ "${target_platform}" == win-* ]]; then
+  /bin/mkdir -p /tmp
+fi
 
-if [ "${UNAME}" == "Linux" ]; then
+if [[ "${target_platform}" == linux-* ]]; then
     export FLIBS="-lgcc_s -lgcc -lstdc++ -lm"
 fi
 
+if [[ "${target_platform}" == osx-arm64 ]]; then
```

Comment:
Can we try without those lines given that you've restored the config changes?

Reply:
just tested locally and it works on `osx-arm64` without this.

Reply:
Sure. I tried to do it in steps just to make sure that the introduction of the file itself does not lead to any failure.
It also seems to have fixed the aarch builds.
Thanks for commit.